lib/validators/dependent.js

```diff
@@ -28,19 +28,22 @@ class Dependent extends Validator {
 
     const FILES_NOT_FOUND_ERROR = `Failed to run the test because 'files' is not provided for 'dependent' option. Please check README for more information about configuration`
     const DEFUALT_SUCCESS_MESSAGE = 'All the Dependents files are represent'
-    let description = validationSettings.message ||
-      'One of the following file is modified, all the other files in the list must be modified as well:' + dependentFiles.map(name => '\n  - ' + name)
 
     // fetch the file list
     let result = await context.github.pullRequests.getFiles(context.repo({number: this.getPayload(context).number}))
-    let modifiedFiles = result.data.filter(file => file.status === 'modified').map(file => file.filename)
-
+    let modifiedFiles = result.data
+      .filter(file => file.status === 'modified' || file.status === 'added')
+      .map(file => file.filename)
     const validatorContext = {name: 'Dependent'}
+
     if (!dependentFiles) {
       return consolidateResult([constructError('Dependent', modifiedFiles, validationSettings, FILES_NOT_FOUND_ERROR)], validatorContext)
     }
     const fileDiff = _.difference(dependentFiles, modifiedFiles)
 
+    let description = validationSettings.message ||
+      `One or two files (${fileDiff.join(', ')}) is missing from your pull request because they are dependent on the following: ${_.difference(dependentFiles, fileDiff)}`
```

Comment:
I think this default message is misleading, i think it would be better to use `One or more files` instead

Reply:
Good catch! I'll update :-)